load-tests/h1_transformation/src/h1_transformation.bal

```diff
@@ -18,56 +18,58 @@ import ballerina/http;
 import ballerina/log;
```

Comment:
Shall we remove log usage from load tests?

Reply:
Fixed with [f771696eb0eff66f228fdeee068f227d6ac91663](https://github.com/ballerina-platform/module-ballerina-http/pull/1166/commits/f771696eb0eff66f228fdeee068f227d6ac91663)